summarizer_server/text_rank.py

```diff
@@ -29,7 +30,10 @@ def _remove_stopwords(self, sen):
 
     # Implemented following:
     #     https://www.analyticsvidhya.com/blog/2018/11/introduction-text-summarization-textrank-python/
-    def summarize(self, input_text):
+    def summarize(self, input_text, percent_sentences):
+        if percent_sentences is None:
```

Comment:
What if `percent_sentences` is > 100? Do we check for this, or is it disallowed on the front end?

A simple workaround for now could be to turn this if statement into `if percent_sentences is None or percent_sentences > 100:`. We can put a `#TODO` in the code, if we want to revisit this later.